OVERVIEW.md

```diff
@@ -60,6 +60,47 @@ You can also pipe input into jscs:
 cat myfile.js | jscs
 ```
 
+## Programmatic Usage
+
+`jscs` can be used directly from your application code:
+
+```js
+var Checker = require("jscs");
+var checker = new Checker();
+```
```

Comment:
Missed something: need to note that we need to call `checker.registerDefaultRules()` to set up all of the built-in rules, otherwise the calls to `.configure` will error out that the rules are undefined